src/frontend/screens/Game/GamePage/index.tsx

```diff
@@ -144,10 +146,21 @@ export default React.memo(function GamePage(): JSX.Element | null {
   useEffect(() => {
     const updateConfig = async () => {
       if (gameInfo) {
-        const { install } = gameInfo
-
-        if (runner !== 'sideload' && !notSupportedGame && install.platform) {
-          getInstallInfo(appName, runner, install.platform)
+        const {
+          install,
+          is_linux_native = undefined,
+          is_mac_native = undefined
+        } = { ...gameInfo }
+
+        const installPlatform =
```

Comment:
Needed to bring back this logic because it was making the getInstallInfo never returns anything. It was previously changed on some recent PR.